docs/topics/proc_creating-reactive-smtp-email-app-webflux.adoc

```diff
@@ -0,0 +1,92 @@
+[id='creating-an-http-smtp-mail-service-with-webflux']
+= Creating a reactive {SpringBoot} SMTP mail app with WebFlux
+
+Create an SMTP email service with {SpringBoot} and WebFlux.
+
+.Prerequisites
+
+* Java JDK 8 or Java JDK 11 installed
+* Maven installed
+* A Maven-based application project configured link:{link-guide-spring-boot}#configuring-your-application-to-use-spring-boot_spring-boot[configured to use Spring Boot]
+// link to postfix config example in RHEL7 Docs
+* A SMTP mail server
+
+.Procedure
+
+. Add `vertx-spring-boot-starter` and `vertx-spring-boot-starter-mail` as dependencies in the `pom.xml` file of your project.
+
+.`pom.xml`
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<project>
+...
+  <dependencies>
+  ...
+include::resources/spring-boot/reactive-examples/example-http-service-mail.pom.xml[tags=http-service-mail-pom-dependencies]
+  ...
+  <dependencies>
+...
+</project>
+----
+
+. Create a mail handler class for your application: 
++
+.src/main/java/dev/snowdrop/vertx/sample/mail/MailHandler.java`
```

Comment:
Source missing

Reply:
Ah, My bad, sorry, I'll make sure to include it, but if you have  plans to re-work the mail example, I can wait:)

Reply:
Added source form the refactored email app sample.